### ISSUE TITLE
appcrash when attempting to run arangod.exe in Windows 7

### ISSUE THREAD

**Issue by Moorfield** [arangodb/arangodb#2911]
I get an appcrash when attempting to install arangoDB in on a windows 7 machine. I have also tried the XCOPY version and have the same issue. The APPCRASH gives the following:

```
 Problem signature:
  Problem Event Name:   APPCRASH
  Application Name: arangod.exe
  Application Version:  0.0.0.0
  Application Timestamp:    59704d12
  Fault Module Name:    arangod.exe
  Fault Module Version: 0.0.0.0
  Fault Module Timestamp:   59704d12
  Exception Code:   c000001d
  Exception Offset: 0000000000200f77
  OS Version:   6.1.7601.2.1.0.256.1
  Locale ID:    2057
  Additional Information 1: caa2
  Additional Information 2: caa2bb545c0b7fee68e5ff27d1b7f78d
  Additional Information 3: 95f8
  Additional Information 4: 95f82d1cb337322ec0f22184a0acdc62
```

I do not believe it even attempts to access the arangod.conf file let alone object to something inside it. I used Windows debugger to try and get some additional clues but I'm left stumped The results are attached.

I have tried running it in safe mode with the same result. I have run chkdsk /f also (I have an ssd drive).

[arangod-windbg.txt](https://github.com/arangodb/arangodb/files/1186826/arangod-windbg.txt)

I'm clutching here but must the disk with arango executable be separate to that running the OS ?
--

**Comment by fceller** [comment]
What kind of processor to you have? Intel or AMD?

**Comment by Moorfield** [comment]
Its an Intel processor. 64 bit Pentium dual core. I eventually resorted to Ubuntu.

**Comment by jsteemann** [comment]
We have released ArangoDB 3.2.1 today. This release potentially fixes the startup crash on Windows.
Can you give it a try and let us know if it is fixed now? Thanks!
